aws-lambda-scorer/README.md

```diff
@@ -110,6 +110,67 @@ The expected response should follow this structure, but the actual values may di
 
 ```json
 {
+  "id": "a12e7390-b8ac-406a-ade9-0d5ea4b63ea9",
+  "score": [
+    [
+      "1.0",
+      "0.6240277982943945",
+      "0.045458571508101536",
+      "0.330513630197504"
+    ],
+    [
+      "3.0",
+      "0.7209441819603676",
+      "0.06299909138586585",
+      "0.21605672665376663"
+    ],
+    [
+      "4.0",
+      "0.7209441819603676",
+      "0.06299909138586585",
+      "0.21605672665376663"
+    ]
+  ]
+}
+```
+
+To get back also the names of fields in the order in which they appear in the resulting `score`,
+pass in an extra request field `includeFieldNames` set to `true`.
```

Comment:
Note: should it be called `includeClassNamesInOutput`?

Reply:
The way it is implemented, it turns names for all the elements in the output. This includes both classes and any fields copied from the input, plus optionally the is field. Wdyt?

Reply:
I'm tempted to say that this should be `true` by default, and `false` only by request... would invite less confusion to first time users, and realistically should not be costly enough where it ever needs to be ignored except in special cases...

Reply:
Thanks @nkpng2k, done.

Reply:
> Note: should it be called `includeClassNamesInOutput`?
> 
Since this change was done, shouldn't the JSON response field name be "class" or "targetClass" instead of "fields"?

Reply:
I implemented the content regarding the default value and tried to explain why the "class" weird does not match. Please see my other comment below :+1:.